lib/mountain_view/engine.rb

```diff
@@ -33,6 +33,7 @@ class Engine < ::Rails::Engine
       ActiveSupport.on_load :action_controller do
         helper MountainView::ApplicationHelper
         helper MountainView::ComponentHelper
+        helper MountainView::StyleguideHelper
```

Comment:
Do we need to include this application-wide? 
It's only used for the styleguide. We can probably put a note in the README that if people want to override the styleguide they can include it themselves.